perun-base/src/main/java/cz/metacentrum/perun/core/api/AssignedMember.java

```diff
@@ -8,20 +8,20 @@
  * @author Johana Supikova <xsupikov@fi.muni.cz>
  */
 public class AssignedMember {
-	private Member member;
+	private RichMember member;
```

Comment:
I would maybe rename this field to richMember and its getter and setter as well, so it is clearer that we are getting / setting RichMember object instead of Member. What do you think?

Reply:
Thanks, renamed. Example added to `parseRpcMethods`.